### PR TITLE
Register warden ares-dev05

### PR DESCRIPTION
**Pull request by code423n4**
Auto-generated PR to register the new warden ares-dev05

@ares-dev05

### CODE REVIEW

**Comment by ares-dev05** [comment]
This is ares-dev05

**Review by ares-dev05 — APPROVED** [review]

**Review by ares-dev05 — APPROVED** [review]

File: _data/handles/ares-dev05.json
Comment (ares-dev05) [inline _data/handles/ares-dev05.json:1]: I checked this json file.

**Review by ares-dev05 — APPROVED** [review]

**Review by ares-dev05 — APPROVED** [review]

**Review by ares-dev05 — COMMENTED** [review]
Checked this changed.

**Review by ares-dev05 — COMMENTED** [review]
Review

**Review by ares-dev05 — APPROVED** [review]

**Review by ares-dev05 — APPROVED** [review]
I checked this json file.

**Review by ares-dev05 — APPROVED** [review]
Review

**Review by ares-dev05 — CHANGES_REQUESTED** [review]
update

**Review by ares-dev05 — APPROVED** [review]

**Review by ares-dev05 — APPROVED** [review]
Reviewed

**Review by ares-dev05 — APPROVED** [review]

**Review by ares-dev05 — CHANGES_REQUESTED** [review]
Merge

**Review by ares-dev05 — CHANGES_REQUESTED** [review]
A

**Review by ares-dev05 — APPROVED** [review]
A

**Review by kartoonjoy — APPROVED** [review]

**Pull request merged**